tests/data/layout1.json

```diff
@@ -30,19 +39,19 @@
     "dpath_tracker_configs": {
         "path": "code/tracker_configs"
```

Comment:
why in code?

Reply:
This is a test file so it really doesn't matter what the paths are called...

Though I think for this case specifically I forgot to rename from `code` to `proc`